components/timeline/assets/Timeline.scss

```diff
@@ -1,86 +1,71 @@
-@import "@not-govuk/sass-base";
+@import "@hods/sass-base";
 @import "govuk-frontend/govuk/helpers/_colour.scss";
 
 .hods-timeline {
   overflow: hidden;
   position: relative;
 
-  p {
-    text-decoration: inherit;
-    font-size: inherit;
-  }
-
   &__item {
     @include govuk-responsive-padding(2, "bottom");
-    @include govuk-responsive-padding(5, "left");
-    /* padding size 6 looks better with smaller viewports. Can media queries
-    be used here? */
+    @media (max-width: 660px) {
+      @include govuk-responsive-padding(7, "left");
+    }
+    @media (min-width: 661px) {
+      @include govuk-responsive-padding(5, "left");
+    }
     position: relative;
 
     &:hover {
-      background-color: #E3E3E3;
+      background-color: govuk-colour("light-grey");
     }
 
-    &:before {
-      background-color: #1d70b8;
+    &:before { /* styling for lines in margin */
+      background-color: govuk-colour("blue");
       content: "";
-      @include govuk-responsive-padding(1, "top"); /* size of blue lines */
+      @include govuk-responsive-padding(1, "top");
       @include govuk-responsive-padding(3, "right");
       left: 0;
       position: absolute;
       @include govuk-responsive-margin(1, "top");
-
       z-index: 1;
     }
   }
 
-  &:before {
-    background-color: #1d70b8;
+  &:before { /* styling for lines in margin */
+    background-color: govuk-colour("blue");
     content: "";
     height: 100%;
     left: 0;
     position: absolute;
-    @include govuk-responsive-margin(1, "top");
-    /* this is more restricive than using px. The ideal margin is around 1.5
-    using this govuk element, but that doesnt exist. I can only use either 1 or
-    2.  1 looks better when the viewport is narrow, and 2 looks better when the
-    viewport is wide. Can @media queries be used here?
-    mobile:  320px,
-    tablet:  641px,
-    desktop: 769px
-    */
+    @media (max-width: 660px) {
+      @include govuk-responsive-margin(1, "top");
+    }
+    @media (min-width: 661px) {
+      @include govuk-responsive-margin(2, "top");
+    }
```

Comment:
You wouldn't use the responsive mixin inside a media query as you would end up with a media query inside a media query.

If you want this level of control you would just set a pixel value inside the media query.

But also I think that GDS provide a mixin for doing these media queries...

Reply:
Something like:
```scss
margin-top: 5px;

@include govuk-media-query($from: mobile) {
  margin-top: 10px;
}
```
I think...

Reply:
@daniel-ac-martin In that case, I think we need to use an absolute value to set the margin here, not a responsive one.
Using a responsive margin causes the element to render incorrectly. When optimised for mobile, it doesnt look good on larger screens. When optimised for desktop, it doesnt look good on mobile. This wasn't a problem when it used a fixed pixel value.

Reply:
I'm saying to _either_ use the responsive mixin or to use the media query mixin but not both.

You could use absolute values as long as they still work on mobile.